packages/create-gatsby/src/index.ts

```diff
@@ -291,42 +293,41 @@ ${c.bold(`Thanks! Here's what we'll now do:`)}
   if (!confirm) {
     trackCli(`CREATE_GATSBY_CANCEL`)
 
-    console.log(`OK, bye!`)
+    reporter.info(`OK, bye!`)
     return
   }
 
   await initStarter(DEFAULT_STARTER, data.project, packages.map(removeKey))
 
-  console.log(
-    c.green(c.symbols.check) + ` Created site in ` + c.green(data.project)
-  )
+  reporter.success(`Created site in ${c.green(data.project)}`)
 
   if (plugins.length) {
-    console.log(c.bold(`${w(`🔌 `)}Installing plugins...`))
+    reporter.info(`${w(`🔌 `)}Setting-up plugins...`)
     await installPlugins(plugins, pluginConfig, path.resolve(data.project), [])
```

Comment:
There was merge conflict here due to change from `path.resolve(data.project)` to `fullPath` added by #28324 which is not backported and was merged before this PR